baal/transformers_trainer_wrapper.py

```diff
@@ -79,7 +79,7 @@ def predict_on_dataset_generator(
             )
 
             out = map_on_tensor(lambda o: o.view([iterations, -1, *o.size()[1:]]), out)
-            out = map_on_tensor(lambda o: o.permute(1, 2, *range(3, o.ndimension()), 0), out)
+            out = map_on_tensor(lambda o: o.permute(1, *range(3, o.ndimension()), 2, 0), out)
```

Comment:
This also makes the result actually match the [documentation](https://github.com/baal-org/baal/pull/263/files#diff-4743e40a145e8d713bf6ff7736bd8e604342f1e3b35d3ad061efa49624f3c199L63)